api/src/controllers/extensions.ts

```diff
@@ -29,6 +30,32 @@ router.get(
 	respond,
 );
 
+router.post(
+	'/:name/:version?',
+	asyncHandler(async (req, _res, next) => {
+		const { name, version } = req.params;
+		const { registry } = req.query;
+
+		if (!name) {
+			throw new ForbiddenError();
+		}
+
+		const authorizationService = new AuthorizationService({
+			accountability: req.accountability,
+			schema: req.schema,
+		});
+
+		//make sure the user can create extensions
+		authorizationService.validatePayload('create', 'directus_extensions', { name, enabled: true });
```

Comment:
is this fine? or should only admins be able to install extensions?

Reply:
I'm leaning towards making the marketplace interactions admin only for the initial launch 👍🏻